src/config/ormconfig.ts

```diff
@@ -1,8 +1,4 @@
 import { DataSourceOptions } from 'typeorm';
-import { config } from 'dotenv';
-
-// Load environment variables from a .env file if necessary
-config();
```

Comment:
Please reconfirm the correct way of loading config from .env.

Reply:
yeah nest.config.ts got this covered